export const replyErrorMessage = async (ctx) => {
    return await ctx.reply("Ответьте на конкретное сообщение")
}
